flixel/system/FlxSound.hx

```diff
@@ -104,7 +104,15 @@ class FlxSound extends FlxBasic
 	/**
 	 * In case of looping, the point (in milliseconds) from where to restart the sound when it loops back
 	 */
-	public var loopTime:Float;
+	public var loopTime(default, set):Float;
+	/**
+	 * In case of looping, the point (in milliseconds) from where to restart the sound when it reaches this time.
+	 */
+	public var loopEndTime(default, set):Float;
```

Comment:
Thinking about this some more, why do we need two separate properties? They don't really seem that different from each other. Wouldn't a single `endTime` be enough, which then handles either the loop point if `looped == true`, or the end point if `looped == false`?

When having two separate properties, users could get into confusing situations, where they use the wrong one (for example `endTime` for a looping sound) and nothing happens.